packages/extension/src/newtab/LinksForm.tsx

```diff
@@ -24,6 +24,12 @@ export function LinksForm({
     setValidInputs((state) => ({ ...state, [i]: isValid }));
   };
 
+  const handleInputChange = (i: number, value: string) => {
+    // Prepend "https://" if the value does not start with "http://" or "https://"
+    const updatedValue = value.startsWith('http://') || value.startsWith('https://') ? value : `https://${value}`;
+    links[i] = updatedValue;
```

Comment:
We're just correcting the link otherwise it's show invalid clicks. and WHY so many reviewers? it's leading to confusion. Please improve your workflow guys!

Reply:
> We're just correcting the link otherwise it'll show invalid link.

Yes, we are correcting the link, but React won't know the changes unless you trigger a state update. This would mean, what is displayed in the UI will be different from what is stored in the updated variable.

> and WHY so many reviewers? it's leading to confusion. Please improve your workflow guys!

At the moment, the app is used by thousands of users, and to ensure workability across the app, we want to make sure whenever we merge something, things would work as intended 🙏